src/ringct/multiexp.cc

```diff
@@ -447,15 +446,14 @@ rct::key straus(const std::vector<MultiexpData> &data, const std::shared_ptr<str
 {
   CHECK_AND_ASSERT_THROW_MES(cache == NULL || cache->size >= data.size(), "Cache is too small");
   MULTIEXP_PERF(PERF_TIMER_UNIT(straus, 1000000));
-  bool HiGi = cache != NULL;
+  bool cache != NULL;
```

Comment:
Looks like this shouldn't even compile.

Reply:
This is overriding the existing variable, so a change is desireable either way.

Reply:
fix it now, remove the whole line.